services/PostService.js

```diff
@@ -0,0 +1,19 @@
+const mongoose      = require('../mongoose');
```

Comment:
You are not using mongoose here, so you should not require it here; rather you should require '../mongoose' in your models, since that's where it is used

Reply:
fixed